benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -198,10 +198,16 @@ void BM_construction(benchmark::State &state, Spec const &spec)
   auto const points =
       constructPoints<DeviceType>(spec.n_values, spec.source_point_cloud_type);
 
+  ExecutionSpace exec_space;
+
   for (auto _ : state)
   {
+    exec_space.fence();
```

Comment:
This fence is unnecessary.

Reply:
I know. The reason I put it there so that if we add some Kokkos things in the beginning in the loop later, they won't affect the timings.